client/landing/gutenboarding/onboarding-block/design-selector/index.tsx

```diff
@@ -66,7 +82,14 @@ const DesignSelector: FunctionComponent = () => {
 							} }
 						>
 							<div className="design-selector__image-frame">
-								<img alt={ design.title } src={ design.src } srcSet={ design.srcset } />
+								<iframe
+									title={ design.title }
+									className="design-selector__frame"
+									src={ getDesignUrl( design, siteTitle ) }
+									scrolling="no"
+									sandbox
```

Comment:
TypeScript complains because this is a string

```suggestion
									sandbox=""
```